src/Budget.php

```diff
@@ -264,7 +264,8 @@ public function rawSearchOptions()
             'table'              => $this->getTable(),
             'field'              => 'value',
             'name'               => _x('price', 'Value'),
-            'datatype'           => 'decimal'
+            'datatype'           => 'decimal',
```

Comment:
```suggestion
            'datatype'           => 'number',
```

Could probably do the trick also.
Not sure on what 'decimal' differs from 'number.
Another possibility is to add the "case 'decimal'" to getActionFor method.

Reply:
IMHO, the lack of `decimal` case in `getActionsFor` is not intentional. It could probably be added in GLPI 10.1.

Anayway, switching to `number` datatype here seems the most relevant option.